pkg/volume/validation/pv_validation.go

```diff
@@ -53,3 +57,15 @@ func checkMountOption(pv *api.PersistentVolume) field.ErrorList {
 	}
 	return allErrs
 }
+
+// ValidatePathNoBacksteps will make sure the targetPath does not have any element which is ".."
+func ValidatePathNoBacksteps(targetPath string) error {
+	parts := strings.Split(targetPath, string(os.PathSeparator))
```

Comment:
Is there anyone from sig-windows that could comment if this is ideal and/or has any impact on existing values?